src/Form.js

```diff
@@ -0,0 +1,78 @@
+import { useState } from "react";
+import orangeLegoTop from "./assets/orangeLegoTop.svg";
+import greenLegoTop from "./assets/greenLegoTop.svg";
+import blueLegoTop from "./assets/blueLegoTop.svg";
+
+const Form = (props) => {
+    const initialInputValues = {
+        pieces: "",
+        year: "",
+    };
+
+    const [inputValues, setInputValues] = useState(initialInputValues);
+
+    const handleChange = (event) => {
+        const { name, value } = event.target;
+
+        setInputValues({
+            ...inputValues,
+            [name]: value,
+        });
+
+    }
+
+    console.log(props.error)
```

Comment:
I'm not sure if you could take out this log? I'm really just trying to find something to comment on